src/vscode-bicep/package.json

```diff
@@ -726,7 +726,25 @@
           }
         ]
       }
-    ]
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "bicep-draggable-resources",
+          "title": "Bicep Resources",
+          "icon": "icons/dumbbell.svg"
```

Comment:
For context, I added the dumbbell icon as a placeholder (and for a bit of fun 😆). We can choose the final icon later.